.gitignore

```diff
@@ -4,6 +4,7 @@ __pycache__/
 *$py.class
 
 # C extensions
+ddtrace/appsec/_ddwaf.cpp
 ddtrace/profiling/collector/_task.c
```

Comment:
If the entire appsec directory is autogenerated, it might make sense to ignore everything in the folder.

Reply:
Seems we have also non-generated files here. I'll see if we can regroup some